# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/crashtest/1849271.html
# Commit: ca737b661c6e
# Full Hash: ca737b661c6eeaecfebf6f2bd27aca46ef656ae4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-08-23 04:21:33
# Description:
#   Bug 1849271 - Flip test when checking for the presence of optional colorspace members on a VideoDecoderConfig. r=media-playback-reviewers,alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D186530
# ==============================================================================

diff -r bc91ede31572 -r ca737b661c6e dom/media/webcodecs/crashtest/1849271.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/media/webcodecs/crashtest/1849271.html	Tue Aug 22 14:33:21 2023 +0000
@@ -0,0 +1,27 @@
+<html class="reftest-wait">
+  <script>
+    var cfg = {
+      codec: "vp8",
+      colorSpace: { primaries: "bt709" },
+    };
+    var decoder = new VideoDecoder({
+      output: () => {},
+      error: e => {
+        document.documentElement.removeAttribute("class");
+      },
+    });
+    decoder.configure(cfg);
+    try {
+      decoder.decode(
+        new EncodedVideoChunk({
+          type: "key",
+          timestamp: 0,
+          duration: 10,
+          data: new Uint8Array(10),
+        })
+      );
+    } catch (e) {
+      document.documentElement.removeAttribute("class");
+    }
+  </script>
+</html>